src/services/cohortCreation/index.ts

```diff
@@ -12,24 +12,32 @@ import {
 import apiFhir from '../api'
 import apiRequest from '../apiRequest'
 import apiBack from '../apiBackCohort'
-import { CONTEXT } from '../../constants'
+import { CONTEXT, FHIR_API_ADMIN_TOKEN, FHIR_API_URL } from '../../constants'
 import { getApiResponseResources } from '../../utils/apiHelpers'
 import type { Cohort_Creation_API_Response, FHIR_API_Response, Query, QueryGroup } from 'types'
+import axios from 'axios'
 
 const PATIENT_MAX_COUNT = 500
 
+const adminApiFhir = axios.create({
+  baseURL: FHIR_API_URL,
+  headers: {
+    Authorization: `Bearer ${FHIR_API_ADMIN_TOKEN}`
```

Comment:
do we need to add this to deployment ?
also, please add #FIXME

Reply:
Nop the env var is already there!